static/compiler.js

```diff
@@ -275,7 +275,8 @@ define(function (require) {
         }, this);
 
         var createCfgView = _.bind(function () {
-            return Components.getCfgViewWith(this.id, this.source, this.lastResult.cfg, this.getCompilerName(), this.sourceEditorId);
+            return Components.getCfgViewWith(this.id, this.source, this.lastResult.cfg, this.getCompilerName(), this.sourceEditorId, 
+                                             this.getEffectiveFilters().binary);
```

Comment:
Again, is this a better approach than asking for the filters once the cfg view has been innited?